examples-standalone/coffee-warehouse/README.md

```diff
@@ -0,0 +1,42 @@
+[<img src="./src/assets/logo-kendo.png" width="225" />](https://www.telerik.com/kendo-angular-ui/) [![Angular Logo](./src/assets/logo-angular.png)](https://angular.io/)
+
+# Using Kendo UI for Angular
+
+This Kendo UI for Angular sample project demonstrates how to use [Kendo UI for Angular components](https://www.telerik.com/kendo-angular-ui/components) within a coffee warehouse prototype application and visualize data. Currently, [the example is deployed on GitHub pages](https://telerik.github.io/kendo-angular/finance-portfolio).
```

Comment:
Wrong URL, also it's not included in [the list](https://github.com/telerik/kendo-angular/blob/e620e46b7a55eb628c4f4049e5856d099bdff0bd/examples/bin/build-gh-pages#L6) of published apps.